src/ansys/fluent/core/workflow.py

```diff
@@ -993,16 +1166,34 @@ def __getattr__(self, attr):
         attr : str
             An attribute not defined in WorkflowWrapper
         """
-        if attr == "add_on_affected":
-            return self._attr_from_wrapped_workflow(attr)
-        return self._task_objects.get(attr) or super().__getattribute__(attr)
+        _task_object = self._task_objects.get(attr)
+        if _task_object:
+            return _task_object
+        if attr != "TaskObject" and attr not in self._unwanted_attrs:
+            if not attr.islower():
+                raise AttributeError(
+                    "Camel case attribute access is not supported. "
+                    f"Try using '{camel_to_snake_case(attr)}' instead."
+                )
+            camel_attr = snake_to_camel_case(str(attr), dir(self._workflow))
+            attr = camel_attr if camel_attr else attr
+            obj = self._attr_from_wrapped_workflow(attr)
+            if obj:
+                return obj
+        return super().__getattribute__(attr)
 
     def __dir__(self):
         """Override the behaviour of dir to include attributes in WorkflowWrapper and
         the underlying workflow."""
```

Comment:
```suggestion
        """Override the behavior of ``dir`` to include attributes in the ``WorkflowWrapper`` class and
        the underlying workflow."""
```
American spellings, please. Tag code entities with double backticks.